clr.coreclr.props

```diff
@@ -83,6 +83,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetsUnix)' != 'true'">
+    <FeatureArrayStubAsIL Condition="('$(TargetArch)' == 'arm') or ('$(TargetArch)' == 'amd64') or ('$(TargetArch)' == 'arm64')">true</FeatureArrayStubAsIL>
```

Comment:
@jkotas Thanks you for comment! It would be much better. 7ec9019 revised PR as you suggested.